BridgeSDKTests/SBBParticipantManagerIntegrationTests.m

```diff
@@ -323,6 +367,33 @@ - (void)testSetExternalIdentifier
             NSLog(@"Timeout setting & checking external identifier: %@", error);
         }
     }];
+    
+    // Now clean up the test externalId so we don't accumulate them over time.
+    // -- First, remove the externalId from the participant:
+    NSDictionary *participant =
+    @{
+      @"externalIds": @{},
+      @"type": @"StudyParticipant"
+      };
+    NSString *participantId = self.testSignInResponseObject[@"id"];
+    SBBParticipantManager *pMan = [SBBParticipantManager managerWithAuthManager:gAdminAuthManager networkManager:SBBComponent(SBBBridgeNetworkManager) objectManager:SBBComponent(SBBObjectManager)];
+    [pMan updateParticipantJSONToBridge:participant completion:^(id  _Nullable responseObject, NSError * _Nullable error) {
+        if (error) {
+            NSLog(@"Failed to remove externalId %@ from test participant: %@", externalIdentifier, error);
+            return;
+        }
+        
+        // -- Now, delete the externalId from the study altogether:
+        NSString *endpoint = [NSString stringWithFormat:@"%@/%@", DEV_EXTERNAL_ID_API, externalIdentifier];
+        [bridgeMan delete:endpoint headers:headers parameters:externalIdObject completion:^(NSURLSessionTask *task, id responseObject, NSError *error) {
+            if (error) {
+                NSLog(@"Failed to delete external ID %@\nError:%@\nResponse:%@", externalIdentifier, error, responseObject);
+                externalIdentifier = nil;
+            } else {
+                NSLog(@"Deleted test external ID %@", externalIdentifier);
+            }
+        }];
+    }];
```

Comment:
Should probably throw an assert if this fails, too. Just so you know that something went wrong.